jobmaster/dm/scheduler/scheduler.go

```diff
@@ -0,0 +1,90 @@
+package scheduler
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"github.com/pingcap/tiflow/dm/pkg/log"
+)
+
+const (
+	checkInterval = 50 * time.Millisecond
+)
+
+type Scheduler interface {
```

Comment:
Is this struct used for source worker scheduling or something? please add some comment

Or we can call it Ticker {Tick} to indicate its calling scenario.